lms/djangoapps/django_comment_client/management/management_utils.py

```diff
@@ -0,0 +1,49 @@
+'''
```

Comment:
hopefully it's not too much more work, but i'd prefer this file to be one more level up, in
`lms/djangoapps/django_comment_client/`